pkg/webhook/inject/inject.go

```diff
@@ -51,8 +60,14 @@ const (
 // Inject do pod template config inject
 func Inject(res *v1.AdmissionRequest, cli client.Client, cfg *config.Config, controllerCfg *controllerCfg.ChaosControllerConfig, metrics *metrics.ChaosControllerManagerMetricsCollector) *v1.AdmissionResponse {
 	var pod corev1.Pod
+	i = Injector{
+		logger:    log.L(),
```

Comment:
I think changes here will lost info : "inject-webhook", in old logger.

Reply:
```suggestion
		logger:    ctrl.Log.WithName("inject-webhook"),
```
What do I need ?

Reply:
It's OK.